### DISCUSSION TITLE
Suggested encoding settings for Opus

### DISCUSSION THREAD

**Discussion by meduar** [pion/webrtc#2590] [Q&A] [answered]
Hi Guys, 
- playing Pion from Linux

I am getting the sound(of a mic in a Linux device) with the command below from an RTSP source, the sound works perfectly in VLC
But when I try to play it on the webrtc-chrome-browser it doesn't work(the video works perfectly), what could it be?

ffmpeg -i rtsp://127.0.0.1:8554/unicast -c:a opus -ac 1 -b:a 64K -vn -strict -2 -f rtp rtp://127.0.0.1:5006

-----------------------------------------------------------------------------------------------
Input #0, rtsp, from 'rtsp://127.0.0.1:8554/unicast':
  Metadata:
    title           : LIVE555 Streaming Media v2023.07.24
    comment         : LIVE555 Streaming Media v2023.07.24
  Duration: N/A, start: 0.000229, bitrate: N/A
    Stream #0:0: Video: h264 (High), yuv420p(progressive), 800x600, 25 tbr, 90k tbn, 180k tbc
    Stream #0:1: Audio: pcm_s16be, 48000 Hz, mono, s16, 768 kb/s
Stream mapping:
  Stream #0:1 -> #0:0 (pcm_s16be (native) -> aac (native))
Press [q] to stop, [?] for help
Output #0, rtp, to 'rtp://127.0.0.1:5006':
  Metadata:
    title           : LIVE555 Streaming Media v2023.07.24
    comment         : LIVE555 Streaming Media v2023.07.24
    encoder         : Lavf58.45.100
    Stream #0:0: Audio: aac (LC), 48000 Hz, mono, fltp, 69 kb/s
    Metadata:
      encoder         : Lavc58.91.100 aac
SDP:
v=0
o=- 0 0 IN IP4 127.0.0.1
s=LIVE555 Streaming Media v2023.07.24
c=IN IP4 127.0.0.1
t=0 0
a=tool:libavformat 58.45.100
m=audio 5006 RTP/AVP 97
b=AS:69
a=rtpmap:97 MPEG4-GENERIC/48000/1
a=fmtp:97 profile-level-id=1;mode=AAC-hbr;sizelength=13;indexlength=3;indexdeltalength=3; config=118856E500

size=    9658kB time=00:19:09.44 bitrate=  68.8kbits/s speed=1.03x
--------------------------------------------------------------------------------------------------

**Comment by Sean-Der** [comment, marked as answer]
Can you try adding a `-page_duration 20000` ?

**Reply by meduar** [reply]
I tried the following none worked:

ffmpeg -i rtsp://127.0.0.1:8554/unicast -c:a opus -ac 1 -b:a 64K -page_duration 20000 -vn -strict -2 -f rtp rtp://127.0.0.1:5006

ffmpeg -i rtsp://127.0.0.1:8554/unicast -c:a aac -ac 1 -b:a 64K -page_duration 20000 -vn -strict -2 -f rtp rtp://127.0.0.1:5006
